cppcache/src/ThinClientLocatorHelper.cpp

```diff
@@ -406,24 +396,24 @@ GfErrType ThinClientLocatorHelper::updateLocators(
 
       auto response =
           std::dynamic_pointer_cast<LocatorListResponse>(di.readObject());
-      auto locators = response->getLocators();
-      if (locators.size() > 0) {
+      auto newLocators = response->getLocators();
+      if (!newLocators.empty()) {
         RandGen randGen;
-        std::random_shuffle(locators.begin(), locators.end(), randGen);
+        std::random_shuffle(newLocators.begin(), newLocators.end(), randGen);
```

Comment:
We would be better served random shuffling every time. You could do us a favor by moving this into your handy-dandy `getLocators` method.

Reply:
Good point 👍